test/unit/specs/validators/email.spec.js

```diff
@@ -1,6 +1,14 @@
 import email from 'src/validators/email'
 
 describe('email validator', () => {
+  it('should validate undefined string', () => {
+    expect(email(undefined)).to.be.true
+  })
+
+  it('should validate null string', () => {
```

Comment:
I think that `should validate null` rather than `should validate null string` is appropriate.